src/components/Forms/SelfScan/index.js

```diff
@@ -48,7 +48,7 @@ export default ({ successMessage, campaignCode }) => {
     if (userId || eMail) {
```

Comment:
Only fetch new data, if there is a specific state, so that it doesn't happen on every render